App/CrisisCleanup/Sources/Feature/Cases/UI/CasesView.swift

```diff
@@ -1,12 +1,46 @@
 import SwiftUI
+import SVGView
 
 struct CasesView: View {
     @Environment(\.translator) var t: KeyAssetTranslator
 
     @ObservedObject var viewModel: CasesViewModel
+    let authenticateViewBuilder: AuthenticateViewBuilder
     let incidentSelectViewBuilder: IncidentSelectViewBuilder
+    let incidentsTypeIconsAssetsFolder = "incident_type_icons/"
 
-    var body: some View {
-        Text(t("casesVue.cases"))
+    @State var showIncidentSelect = false
+
+    var body: some View{
+        VStack {
+            HStack {
+                Button {
+                    showIncidentSelect.toggle()
+                } label: {
+                    HStack{
```

Comment:
The `HStack` is not necessary. There's only one view inside it so the `HStack` container is redundant.